### PR TITLE
Update terminology.mdx

### PR DESCRIPTION
**Pull request by Kelechikizito**
I updated the terminology docs

### CODE REVIEW

**Comment by Kelechikizito** [comment]
@jthiller please add the label and review, thanks.

File: docs/home/faq/terminology.mdx [hunk 1]
```diff
@@ -23,6 +23,9 @@ slug: /faq/terminology
 - **Developers** - Individuals or organizations that build applications and services using the
   Helium Network infrastructure.
 
+- **DePIN** - DePIN, or Decentralized Physical Infrastructure Networks, projects use decentralized technology to power   
+  real-world uses, such as street maps, rendering power, cab-hailing services, and more.
```
Comment (jthiller) [inline docs/home/faq/terminology.mdx]: The definition should probably include wireless networks since we're in the Helium docs.

Comment (Kelechikizito) [reply docs/home/faq/terminology.mdx]: I have included wireless networks.

**Review by jthiller — APPROVED** [review]

**Comment by jthiller** [comment]
Thank you for submitting updates to the Helium Docs as part of the Solana Scribes Hackathon! Please follow the instructions in this form so we may begin issuing awards.

https://forms.gle/z4ovMHYDo7pzy8TN6  

Please complete this form by March 15, 2024. (Approximately 1 week from today). We had SO MANY great submissions and we’re eager to distribute the awards to all of the contributors.

**Comment by jthiller** [comment]
Apologies, something happened to the Google forms link. This one should work: [https://forms.gle/z4ovMHYDo7pzy8TN6](https://forms.gle/z4ovMHYDo7pzy8TN6)

**Comment by Kelechikizito** [comment]
Confirmation phrase:
wireless

**Pull request merged**